docs/user-guide.md

```diff
@@ -61,6 +61,7 @@ Usage of kube-router:
       --ipvs-graceful-period duration                 The graceful period before removing destinations from IPVS services (e.g. '5s', '1m', '2h22m'). Must be greater than 0. (default 30s)
       --ipvs-graceful-termination                     Enables the experimental IPVS graceful terminaton capability
       --ipvs-sync-period duration                     The delay between ipvs config synchronizations (e.g. '5s', '1m', '2h22m'). Must be greater than 0. (default 5m0s)
+      --ipvs-allow-all-input                          Enables rule to accept all incoming traffic on the node. (default true).
```

Comment:
prefer --ipvs-deny-all which defaults to false

Reply:
@andrewsykim We may be better off just not applying the chain based on my testing. I am spending a little more time on this just to make sure it does exactly what's expected and has no adverse negative impact.

Reply:
thanks, appreciate the thoroughness